openapi_test.go

```diff
@@ -15,10 +15,17 @@ import (
 
 type MyStruct struct {
 	B string `json:"b"`
-	C int    `json:"c"`
+	C int    `json:"c" example:"8" validate:"min=3,max=10" description:"my description"`
 	D bool   `json:"d"`
 }
 
+type MyStructWithNested struct {
+	E      string   `json:"e" example:"E"`
+	F      int      `json:"f"`
+	G      bool     `json:"g"`
+	Nested MyStruct `json:"nested" description:"my struct"`
```

Comment:
I'm curious about what would bring a embedded struct

So please add something comparable to this to the examples or tests

```suggestion
	Nested MyStruct `json:"nested" description:"my struct"`
	MyStruct
```

I'm not expecting to fail, but I would prefer to see it covered by a test, this way any future refactoring would be safer

Please note, I have no idea whether it's already covered by an existing unit test. If so, then my remark is pointless maybe

Reply:
added